sysvabi64/sysvabi64.rst

```diff
@@ -1643,11 +1648,25 @@ Its use is optional, meaning that an ELF file where this feature bit
 is unset can still have Return Address signing enabled in some or all of
 its executable sections.
 
+``GNU_PROPERTY_AARCH64_FEATURE_1_GCS`` This indicates that all
+executable sections are compatible with the Guarded control stack
+(GCS) mechanism. Minimum requirements for setting this feature bit
+include:
+
+* The number of ``procedure return address push operations`` and the
```

Comment:
The terms "procedure return address push operation" and "procedure return address pop operation" are defined only in the GCS specification (as far as I know) and are perhaps a bit misleading here (on first reading I thought it was talking about the STP at function start and LDP at function end, but it's actually talking about BL and RET). Also having a matching number of BL/RET isn't enough, every RET has to match up with the BL that called the function.

I think rewording this to be something like the following would be clearer:
> Each function that is called using a BL instruction (or other instruction that is a GCS procedure return address push operation) returns using a RET instruction (or other instruction that is a GSC procedure return address pop operation). This means that RET instructions are used for function returns, and not as an indirect branch.

Reply:
Thanks for the suggestion, I've used an almost identical wording in a fixup commit that I'll squash prior to merging.